Documentation/dev-guide/interacting_v3.md

```diff
@@ -116,6 +141,28 @@ bar
 $ etcdctl get --rev=1 foo foo9 # access the versions of keys at revision 1
 ```
 
+## Read keys which are greater than or equal to the byte value of the specified key
+
+Applications may want to Read keys which are greater than or equal to the byte value of the specified key.
```

Comment:
@xiang90 in my local version its showing small with nothing to commit or push. Will check it and fix.
Thanks!